js/app/test/gallery_component_events.spec.ts

```diff
@@ -18,6 +18,7 @@ test("Gallery preview mode displays all images correctly.", async ({
 test("Gallery select event returns the right value", async ({ page }) => {
 	await page.getByRole("button", { name: "Run" }).click();
 	await page.getByLabel("Thumbnail 2 of 3").click();
+	await page.waitForTimeout(200);
```

Comment:
So for some reason, playwright immediately says the value is `""`. Any thoughts why @pngwn ?

Adding a brief pause, or setting `queue=False` in the share event seems to make the problem go away.

Reply:
Also weird cause I swear this wasn't flaking when we first wrote this